providers.tf.example

```diff
@@ -21,7 +22,11 @@ provider "helm" {
     host                   = data.aws_eks_cluster.cluster.endpoint
     cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority[0].data)
     token                  = data.aws_eks_cluster_auth.cluster.token
+    exec {
+      api_version = "client.authentication.k8s.io/v1beta1"
+      args        = ["eks", "--profile=change_me", "get-token", "--cluster-name", data.aws_eks_cluster.cluster.name]
```

Comment:
I was not able to reproduce the error, but I was not using WSL but an ubuntu docker container. In terms of this PR I suggest to remove the exec part but we may come back to this later if we encounter the problem again.

Reply:
For sure it did not work for me on WSL2, helm deployments would time out. Implementing exec command would obtain a fresh token and exact same charts would then be successfully deployed.

In any case, I will remove it for now, but we need to keep this in mind, if we ran into compatibility issues later on, as you mention.

Reply:
Erm, commented out would work as well, no? We have a ready solution in case someone runs into this problem, and since it's commented out, it won't affect the rest - a win-win.

Reply:
I agree